src/liquid/components/ld-slider/ld-slider.tsx

```diff
@@ -94,7 +94,9 @@ export class LdSlider implements InnerFocusable {
   /** Adds custom stop points to the slider (instead of steps) */
   @Prop() unit?: string
   /** Specifies the default value */
-  @Prop({ mutable: true, reflect: true }) value?: string = String(this.min)
+  @Prop({ mutable: true, reflect: true }) value?: string | number = String(
+    this.min
+  )
```

Comment:
```suggestion
  @Prop({ mutable: true, reflect: true }) value?: string | number = this.min
```

Wouldn't this work as well, then?